build.gradle

```diff
@@ -48,6 +48,7 @@ allprojects {
     repositories {
         jcenter()
         maven { url 'https://kotlin.bintray.com/kotlinx' }
+        maven { url 'https://dl.bintray.com/badoo/maven' }
```

Comment:
And please, Arkadii, add this repository just for the new module in a `repositories` section at the end of `modules/fx/arrow-fx-reaktive/build.gradle`. Thanks!!